### ISSUE TITLE
[Baidu Search API] Support `shopping_results` for desktop

### ISSUE THREAD

**Issue by zyc9012** [serpapi/public-roadmap#523]
<img width="616" alt="image" src="https://user-images.githubusercontent.com/3034310/205007518-263d7942-118c-4f33-a9a6-7480a124d0c3.png">

[Inspect](https://serpapi.com/searches/638869393e86acfa7db2fd97/inspect)

Currently, this block is incorrectly parsed into `answer_box`

**Comment by hartator** [comment]
Do we have customers asking for all these Baidu results?

**Comment by zyc9012** [comment]
> Do we have customers asking for all these Baidu results?

No. But we have supported these results on the mobile part, I think we can do it on the desktop as well.

**Comment by hartator** [comment]
> No. But we have supported these results on the mobile part, I think we can do it on the desktop as well.

Maybe let's wait for someone asking for it? It's not good if we are working on APIs that nobody wants.
